api/views/auth.py

```diff
@@ -65,55 +58,56 @@ def create_account():
     response_data.pop('pw_hash')
     response_data.pop('salt')
     response_data['token'] = token
-    return create_response(data=response_data, status=201,
-                            message='successfully created account')
+    return create_response(data=response_data, status=201, 
+            message='Successfully created account')

```

Comment:
this spacing is making me feel weird lol :3 
if it's too long on one line, can we make each param its own line? 
also applies to rest of PR

Reply:
🆗 👍